src/postman/transformers/request.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  HttpParamStyles,
+  IHttpHeaderParam,
+  IHttpOperationRequestBody,
+  IHttpPathParam,
+  IHttpQueryParam,
+  IMediaTypeContent,
+} from '@stoplight/types';
+import { JSONSchema4 } from 'json-schema';
+import { FormParam, HeaderDefinition, PropertyList, QueryParam, RequestBody } from 'postman-collection';
+import { transformDescriptionDefinition, transformPostmanTemplate, transformValueToHttpParam } from '../util';
+
+export function transformQueryParam(queryParam: QueryParam): IHttpQueryParam {
+  return {
+    name: queryParam.key as string, // no key no game
```

Comment:
This can effectively be null according to the spec, I am afraid we can't really get it in this way.

The Postman collection apparently translates/sends to the network a null parameter as an empty string — we should be able to do that here as well, although we might need to group them

```suggestion
    name: queryParam.key || ''
```

Reply:
I checked in Postman App: when I make key of query param an empty string, it is not added to the request at all. It is also exported as empty string.

Thinking about two things:
1. leave a param empty-string named
2. remove those empty-string named params

Quite hard for me to make a decision:
- it can be useful for someone to keep the information about those params (e.g. one can fill missing information in our eco-system)
- keeping empty-named params will make then unusable in that form? it may cause problems with creating requests or mocking?

@philsturgeon @XVincentX WDYT?